### DISCUSSION TITLE
I can't get it to work

### DISCUSSION THREAD

**Discussion by Linkanite** [cangzhang/ChampR#301] [Q&A]
![image](https://github.com/cangzhang/champ-r/assets/116902520/039e3ffe-b78b-4190-89ea-fa48604ef6a5)

idk why

**Comment by cangzhang** [comment]
Hi, our server was temporarily down at that moment, it's back online a few hours ago, please try again

**Reply by Lz1y** [reply]
至今还是不行，源码中的服务器请求返回状态码500，是服务器问题还没解决吗？
https://github.com/cangzhang/champ-r/blob/47fe3110b8a311a693cc310d12f20e3b6b751559/src/web.rs#L64C37-L64C37
